pom.xml

```diff
@@ -309,6 +309,15 @@
                     <groupId>net.revelc.code</groupId>
                     <artifactId>impsort-maven-plugin</artifactId>
                     <version>1.5.0</version>
+                    <dependencies>
+                        <!-- Needed because of maven 3.9.0 backward compatibility issue,
+                             See : https://github.com/revelc/impsort-maven-plugin/issues/64 -->
+                        <dependency>
+                            <groupId>org.codehaus.plexus</groupId>
+                            <artifactId>plexus-utils</artifactId>
+                            <version>3.5.1</version>
+                        </dependency>
+                    </dependencies>
```

Comment:
The changes here are also changes we should make to the archetype for modules.

Reply:
noted!, will make a commit to the sdk for it